docs/api-reference/carto/carto-sql-layer.md

```diff
@@ -48,7 +74,7 @@ To use pre-bundled scripts:
 <script src="https://unpkg.com/@deck.gl/carto@^8.2.0/dist.min.js"></script>
```

Comment:
@^8.5.0 is used in previous examples. I guess you should be homogeneous here with it